bug: Can't start on linux (Wayland)

### Jan version

0.6

### Describe the Bug

When I launch the appImage, it doesn't start and crashes.

```
 Downloads ./Jan_0.6.0_amd64.AppImage
[2025-06-19][09:40:39][app_lib::core::setup][INFO] Installing extensions. Clean up: false, Stored version: 0.6.0, App version: 0.6.0
[2025-06-19][09:40:39][app_lib::core::setup][ERROR] Failed to run mcp commands: Failed to read config file: No such file or directory (os error 2)
[2025-06-19][09:40:39][app_lib::core::setup][INFO] Clean up function executed, sidecar processes killed.
[2025-06-19][09:40:39][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries" to "/home/peyloride/jan"
[2025-06-19][09:40:39][app_lib::core::setup][INFO] Spawning cortex-server (Attempt 1/5)
[2025-06-19][09:40:39][app_lib::core::setup][DEBUG] Sidecar command: Command { cmd: LD_LIBRARY_PATH="/tmp/.mount_Jan_0.i4BXwa/usr/lib/:/tmp/.mount_Jan_0.i4BXwa/usr/lib/i386-linux-gnu/:/tmp/.mount_Jan_0.i4BXwa/usr/lib/x86_64-linux-gnu/:/tmp/.mount_Jan_0.i4BXwa/usr/lib32/:/tmp/.mount_Jan_0.i4BXwa/usr/lib64/:/tmp/.mount_Jan_0.i4BXwa/lib/:/tmp/.mount_Jan_0.i4BXwa/lib/i386-linux-gnu/:/tmp/.mount_Jan_0.i4BXwa/lib/x86_64-linux-gnu/:/tmp/.mount_Jan_0.i4BXwa/lib32/:/tmp/.mount_Jan_0.i4BXwa/lib64/::/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries" "/tmp/.mount_Jan_0.i4BXwa/usr/bin/cortex-server" "--start-server" "--port" "39291" "--config_file_path" "/home/peyloride/jan/.janrc" "--data_folder_path" "/home/peyloride/jan" "--cors" "ON" "--allowed_origins" "http://localhost:3000,http://localhost:1420,tauri://localhost,http://tauri.localhost" "config" "--api_keys" "ON1X9G0OnOr0TKVwmzP1iPRWFw3qfan7", raw_out: false }
[2025-06-19][09:40:39][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries/engines" to "/home/peyloride/jan/engines"
[2025-06-19][09:40:39][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries/engines/llama.cpp" to "/home/peyloride/jan/engines/llama.cpp"
[2025-06-19][09:40:39][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx-x64" to "/home/peyloride/jan/engines/llama.cpp/linux-avx-x64"
[2025-06-19][09:40:39][app_lib::core::setup][INFO] Cortex server spawned successfully. PID: 38760
[2025-06-19][09:40:39][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx-x64/b5509" to "/home/peyloride/jan/engines/llama.cpp/linux-avx-x64/b5509"
[2025-06-19][09:40:39][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-cuda-cu11.7-x64" to "/home/peyloride/jan/engines/llama.cpp/linux-avx2-cuda-cu11.7-x64"
[2025-06-19][09:40:39][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-cuda-cu11.7-x64/b5509" to "/home/peyloride/jan/engines/llama.cpp/linux-avx2-cuda-cu11.7-x64/b5509"
[2025-06-19][09:40:39][app_lib::core::setup][INFO] [Cortex Terminated]: Signal Some(15), Code None
[2025-06-19][09:40:39][app_lib::core::setup][INFO] Cleared child process lock after termination.
[2025-06-19][09:40:39][app_lib::core::setup][WARN] Cortex server terminated unexpectedly.
[2025-06-19][09:40:39][app_lib::core::setup][INFO] Waiting 5000ms before attempting restart 1/5...
[2025-06-19][09:40:39][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-cuda-cu12.0-x64" to "/home/peyloride/jan/engines/llama.cpp/linux-avx2-cuda-cu12.0-x64"
[2025-06-19][09:40:39][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-cuda-cu12.0-x64/b5509" to "/home/peyloride/jan/engines/llama.cpp/linux-avx2-cuda-cu12.0-x64/b5509"
[2025-06-19][09:40:40][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-x64" to "/home/peyloride/jan/engines/llama.cpp/linux-avx2-x64"
[2025-06-19][09:40:40][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-x64/b5509" to "/home/peyloride/jan/engines/llama.cpp/linux-avx2-x64/b5509"
[2025-06-19][09:40:40][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx512-x64" to "/home/peyloride/jan/engines/llama.cpp/linux-avx512-x64"
[2025-06-19][09:40:40][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx512-x64/b5509" to "/home/peyloride/jan/engines/llama.cpp/linux-avx512-x64/b5509"
[2025-06-19][09:40:40][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-cuda-cu11.7-x64" to "/home/peyloride/jan/engines/llama.cpp/linux-noavx-cuda-cu11.7-x64"
[2025-06-19][09:40:40][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-cuda-cu11.7-x64/b5509" to "/home/peyloride/jan/engines/llama.cpp/linux-noavx-cuda-cu11.7-x64/b5509"
[2025-06-19][09:40:40][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-cuda-cu12.0-x64" to "/home/peyloride/jan/engines/llama.cpp/linux-noavx-cuda-cu12.0-x64"
[2025-06-19][09:40:40][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-cuda-cu12.0-x64/b5509" to "/home/peyloride/jan/engines/llama.cpp/linux-noavx-cuda-cu12.0-x64/b5509"
[2025-06-19][09:40:41][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-x64" to "/home/peyloride/jan/engines/llama.cpp/linux-noavx-x64"
[2025-06-19][09:40:41][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-x64/b5509" to "/home/peyloride/jan/engines/llama.cpp/linux-noavx-x64/b5509"
[2025-06-19][09:40:41][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries/engines/llama.cpp/linux-vulkan-x64" to "/home/peyloride/jan/engines/llama.cpp/linux-vulkan-x64"
[2025-06-19][09:40:41][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries/engines/llama.cpp/linux-vulkan-x64/b5509" to "/home/peyloride/jan/engines/llama.cpp/linux-vulkan-x64/b5509"
[2025-06-19][09:40:43][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/resources" to "/home/peyloride/jan"
[2025-06-19][09:40:43][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/resources/pre-install" to "/home/peyloride/jan/pre-install"
Could not create default EGL display: EGL_BAD_PARAMETER. Aborting...
[2025-06-19][09:40:44][app_lib::core::setup][INFO] Spawning cortex-server (Attempt 2/5)
[2025-06-19][09:40:44][app_lib::core::setup][DEBUG] Sidecar command: Command { cmd: LD_LIBRARY_PATH="/tmp/.mount_Jan_0.i4BXwa/usr/lib/:/tmp/.mount_Jan_0.i4BXwa/usr/lib/i386-linux-gnu/:/tmp/.mount_Jan_0.i4BXwa/usr/lib/x86_64-linux-gnu/:/tmp/.mount_Jan_0.i4BXwa/usr/lib32/:/tmp/.mount_Jan_0.i4BXwa/usr/lib64/:/tmp/.mount_Jan_0.i4BXwa/lib/:/tmp/.mount_Jan_0.i4BXwa/lib/i386-linux-gnu/:/tmp/.mount_Jan_0.i4BXwa/lib/x86_64-linux-gnu/:/tmp/.mount_Jan_0.i4BXwa/lib32/:/tmp/.mount_Jan_0.i4BXwa/lib64/::/tmp/.mount_Jan_0.i4BXwa/usr/lib/Jan/binaries" "/tmp/.mount_Jan_0.i4BXwa/usr/bin/cortex-server" "--start-server" "--port" "39291" "--config_file_path" "/home/peyloride/jan/.janrc" "--data_folder_path" "/home/peyloride/jan" "--cors" "ON" "--allowed_origins" "http://localhost:3000,http://localhost:1420,tauri://localhost,http://tauri.localhost" "config" "--api_keys" "ON1X9G0OnOr0TKVwmzP1iPRWFw3qfan7", raw_out: false }
[2025-06-19][09:40:44][app_lib::core::setup][INFO] Cortex server spawned successfully. PID: 38824
[2025-06-19][09:40:44][app_lib::core::setup][INFO] Cortex server started successfully, resetting restart count from 1 to 0.
```

### Steps to Reproduce

_No response_

### Screenshots / Logs

_No response_

### What is your OS?

- [ ] MacOS
- [ ] Windows
- [x] Linux

Can confirm - AppImage not working as expected on Wayland linux (for me it starts, but window has no content and stays like that).
Info:
<details><summary>Log</summary>
<p>
./Jan_0.6.0_amd64.AppImage 

```
(process:11336): Gtk-WARNING **: 13:26:44.044: Locale not supported by C library.
	Using the fallback 'C' locale.
[2025-06-19][11:26:44][app_lib::core::setup][INFO] Installing extensions. Clean up: false, Stored version: 0.6.0, App version: 0.6.0
[2025-06-19][11:26:44][app_lib::core::setup][ERROR] Failed to run mcp commands: Failed to read config file: No such file or directory (os error 2)
[2025-06-19][11:26:44][app_lib::core::setup][INFO] Clean up function executed, sidecar processes killed.
[2025-06-19][11:26:44][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries" to "/home/fenriv/.local/share/Jan/data"
[2025-06-19][11:26:44][app_lib::core::setup][INFO] Spawning cortex-server (Attempt 1/5)
[2025-06-19][11:26:44][app_lib::core::setup][DEBUG] Sidecar command: Command { cmd: LD_LIBRARY_PATH="/tmp/.mount_Jan_0.i0Vc2e/usr/lib/:/tmp/.mount_Jan_0.i0Vc2e/usr/lib/i386-linux-gnu/:/tmp/.mount_Jan_0.i0Vc2e/usr/lib/x86_64-linux-gnu/:/tmp/.mount_Jan_0.i0Vc2e/usr/lib32/:/tmp/.mount_Jan_0.i0Vc2e/usr/lib64/:/tmp/.mount_Jan_0.i0Vc2e/lib/:/tmp/.mount_Jan_0.i0Vc2e/lib/i386-linux-gnu/:/tmp/.mount_Jan_0.i0Vc2e/lib/x86_64-linux-gnu/:/tmp/.mount_Jan_0.i0Vc2e/lib32/:/tmp/.mount_Jan_0.i0Vc2e/lib64/::/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries" "/tmp/.mount_Jan_0.i0Vc2e/usr/bin/cortex-server" "--start-server" "--port" "39291" "--config_file_path" "/home/fenriv/.local/share/Jan/data/.janrc" "--data_folder_path" "/home/fenriv/.local/share/Jan/data" "--cors" "ON" "--allowed_origins" "http://localhost:3000,http://localhost:1420,tauri://localhost,http://tauri.localhost" "config" "--api_keys" "8NBRhZfv6Ww3RbMLP7R0FOZ7qLTqATbN", raw_out: false }
[2025-06-19][11:26:44][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries/engines" to "/home/fenriv/.local/share/Jan/data/engines"
[2025-06-19][11:26:44][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries/engines/llama.cpp" to "/home/fenriv/.local/share/Jan/data/engines/llama.cpp"
[2025-06-19][11:26:44][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx-x64" to "/home/fenriv/.local/share/Jan/data/engines/llama.cpp/linux-avx-x64"
[2025-06-19][11:26:44][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx-x64/b5509" to "/home/fenriv/.local/share/Jan/data/engines/llama.cpp/linux-avx-x64/b5509"
[2025-06-19][11:26:44][app_lib::core::setup][INFO] Cortex server spawned successfully. PID: 11454
[2025-06-19][11:26:44][app_lib::core::setup][INFO] [Cortex Terminated]: Signal Some(15), Code None
[2025-06-19][11:26:44][app_lib::core::setup][INFO] Cleared child process lock after termination.
[2025-06-19][11:26:44][app_lib::core::setup][WARN] Cortex server terminated unexpectedly.
[2025-06-19][11:26:44][app_lib::core::setup][INFO] Waiting 5000ms before attempting restart 1/5...
[2025-06-19][11:26:44][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-cuda-cu11.7-x64" to "/home/fenriv/.local/share/Jan/data/engines/llama.cpp/linux-avx2-cuda-cu11.7-x64"
[2025-06-19][11:26:44][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-cuda-cu11.7-x64/b5509" to "/home/fenriv/.local/share/Jan/data/engines/llama.cpp/linux-avx2-cuda-cu11.7-x64/b5509"

(process:11435): Gtk-WARNING **: 13:26:44.280: Locale not supported by C library.
	Using the fallback 'C' locale.
[2025-06-19][11:26:45][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-cuda-cu12.0-x64" to "/home/fenriv/.local/share/Jan/data/engines/llama.cpp/linux-avx2-cuda-cu12.0-x64"
[2025-06-19][11:26:45][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-cuda-cu12.0-x64/b5509" to "/home/fenriv/.local/share/Jan/data/engines/llama.cpp/linux-avx2-cuda-cu12.0-x64/b5509"
[2025-06-19][11:26:46][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-x64" to "/home/fenriv/.local/share/Jan/data/engines/llama.cpp/linux-avx2-x64"
[2025-06-19][11:26:46][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-x64/b5509" to "/home/fenriv/.local/share/Jan/data/engines/llama.cpp/linux-avx2-x64/b5509"
[2025-06-19][11:26:46][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx512-x64" to "/home/fenriv/.local/share/Jan/data/engines/llama.cpp/linux-avx512-x64"
[2025-06-19][11:26:46][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx512-x64/b5509" to "/home/fenriv/.local/share/Jan/data/engines/llama.cpp/linux-avx512-x64/b5509"
[2025-06-19][11:26:46][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-cuda-cu11.7-x64" to "/home/fenriv/.local/share/Jan/data/engines/llama.cpp/linux-noavx-cuda-cu11.7-x64"
[2025-06-19][11:26:46][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-cuda-cu11.7-x64/b5509" to "/home/fenriv/.local/share/Jan/data/engines/llama.cpp/linux-noavx-cuda-cu11.7-x64/b5509"
[2025-06-19][11:26:47][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-cuda-cu12.0-x64" to "/home/fenriv/.local/share/Jan/data/engines/llama.cpp/linux-noavx-cuda-cu12.0-x64"
[2025-06-19][11:26:47][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-cuda-cu12.0-x64/b5509" to "/home/fenriv/.local/share/Jan/data/engines/llama.cpp/linux-noavx-cuda-cu12.0-x64/b5509"
[2025-06-19][11:26:48][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-x64" to "/home/fenriv/.local/share/Jan/data/engines/llama.cpp/linux-noavx-x64"
[2025-06-19][11:26:48][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-x64/b5509" to "/home/fenriv/.local/share/Jan/data/engines/llama.cpp/linux-noavx-x64/b5509"
[2025-06-19][11:26:48][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries/engines/llama.cpp/linux-vulkan-x64" to "/home/fenriv/.local/share/Jan/data/engines/llama.cpp/linux-vulkan-x64"
[2025-06-19][11:26:48][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries/engines/llama.cpp/linux-vulkan-x64/b5509" to "/home/fenriv/.local/share/Jan/data/engines/llama.cpp/linux-vulkan-x64/b5509"
[2025-06-19][11:26:49][app_lib::core::setup][INFO] Spawning cortex-server (Attempt 2/5)
[2025-06-19][11:26:49][app_lib::core::setup][DEBUG] Sidecar command: Command { cmd: LD_LIBRARY_PATH="/tmp/.mount_Jan_0.i0Vc2e/usr/lib/:/tmp/.mount_Jan_0.i0Vc2e/usr/lib/i386-linux-gnu/:/tmp/.mount_Jan_0.i0Vc2e/usr/lib/x86_64-linux-gnu/:/tmp/.mount_Jan_0.i0Vc2e/usr/lib32/:/tmp/.mount_Jan_0.i0Vc2e/usr/lib64/:/tmp/.mount_Jan_0.i0Vc2e/lib/:/tmp/.mount_Jan_0.i0Vc2e/lib/i386-linux-gnu/:/tmp/.mount_Jan_0.i0Vc2e/lib/x86_64-linux-gnu/:/tmp/.mount_Jan_0.i0Vc2e/lib32/:/tmp/.mount_Jan_0.i0Vc2e/lib64/::/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/binaries" "/tmp/.mount_Jan_0.i0Vc2e/usr/bin/cortex-server" "--start-server" "--port" "39291" "--config_file_path" "/home/fenriv/.local/share/Jan/data/.janrc" "--data_folder_path" "/home/fenriv/.local/share/Jan/data" "--cors" "ON" "--allowed_origins" "http://localhost:3000,http://localhost:1420,tauri://localhost,http://tauri.localhost" "config" "--api_keys" "8NBRhZfv6Ww3RbMLP7R0FOZ7qLTqATbN", raw_out: false }
[2025-06-19][11:26:49][app_lib::core::setup][INFO] Cortex server spawned successfully. PID: 11462
[2025-06-19][11:26:49][app_lib::core::setup][INFO] Cortex server started successfully, resetting restart count from 1 to 0.
[2025-06-19][11:26:50][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/resources" to "/home/fenriv/.local/share/Jan/data"
[2025-06-19][11:26:50][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.i0Vc2e/usr/lib/Jan/resources/pre-install" to "/home/fenriv/.local/share/Jan/data/pre-install"
Could not create default EGL display: EGL_BAD_PARAMETER. Aborting...
```

</p>
</details> 



<details><summary>System info</summary>
<p>

```
Bazzite 42 (FROM Fedora Kinoite)
Linux 6.14.6-106.bazzite.fc42.x86_64
AMD Ryzen 7 5800X3D (16) @ 4.55 GHz
AMD RX 9070/9070 XT [Discrete]
6.48 GiB / 31.24 GiB (21%)
41.43 MiB / 41.43 MiB (100%) - overlay [Read-only]
340.45 GiB / 1.36 TiB (24%) - btrfs [Read-only]
2560x1440 @ 144 Hz in 27" [External]
     KDE Plasma 6.3.5
     KWin (Wayland)
```

</p>
</details>

> This might be a [problem related to WebKit](https://github.com/zmkfirmware/zmk-studio/issues/83).
> 
> According to the issue linked above, this might help:
> 
> ```
> WEBKIT_DISABLE_COMPOSITING_MODE=1 WEBKIT_DISABLE_DMABUF_RENDERER=1 ./Jan_0.6.1_amd64.appimage
> ```
> 
> If it fixes the problem, you may want to try just one of them (the first one helped with a problem I had some time ago) and report if it worked.
> 
> If not, to help troubleshooting, probably reporting this info will help:
> 
>     1. Desktop Environment with version
> 
>     2. Wayland or X11
> 
>     3. GPU
> 
>     4. Check if gtk4 and/or webkit2gtk-4.x is installed

Not a problem with webkit,it starts with black screen and then its transparent,it does not crash.
Info:

- 1.KDE 6.4
- 2.Wayland
- 3.Amd rx 6800 xt
- 4.yes and no,even with webkit,nothing changed.

Everything was fine on 5.7,but 6.1 and beta releases are broken.

Here is my logs : 


> ./Jan_0.6.1_amd64.AppImage
> [2025-06-20][21:26:15][app_lib::core::setup][INFO] Installing extensions. Clean up: false, Stored version: 0.6.1, App version: 0.6.1
> [2025-06-20][21:26:15][app_lib::core::setup][ERROR] Failed to run mcp commands: Failed to read config file: No such file or directory (os error 2)
> [2025-06-20][21:26:15][app_lib::core::setup][INFO] Clean up function executed, sidecar processes killed.
> [2025-06-20][21:26:15][app_lib::core::setup][INFO] Spawning cortex-server (Attempt 1/5)
> [2025-06-20][21:26:15][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries" to "/home/yopi/.config/Jan/data"
> [2025-06-20][21:26:15][app_lib::core::setup][DEBUG] Sidecar command: Command { cmd: LD_LIBRARY_PATH="/tmp/.mount_Jan_0.xnvX1c/usr/lib/:/tmp/.mount_Jan_0.xnvX1c/usr/lib/i386-linux-gnu/:/tmp/.mount_Jan_0.xnvX1c/usr/lib/x86_64-linux-gnu/:/tmp/.mount_Jan_0.xnvX1c/usr/lib32/:/tmp/.mount_Jan_0.xnvX1c/usr/lib64/:/tmp/.mount_Jan_0.xnvX1c/lib/:/tmp/.mount_Jan_0.xnvX1c/lib/i386-linux-gnu/:/tmp/.mount_Jan_0.xnvX1c/lib/x86_64-linux-gnu/:/tmp/.mount_Jan_0.xnvX1c/lib32/:/tmp/.mount_Jan_0.xnvX1c/lib64/::/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries" "/tmp/.mount_Jan_0.xnvX1c/usr/bin/cortex-server" "--start-server" "--port" "39291" "--config_file_path" "/home/yopi/.config/Jan/data/.janrc" "--data_folder_path" "/home/yopi/.config/Jan/data" "--cors" "ON" "--allowed_origins" "http://localhost:3000,http://localhost:1420,tauri://localhost,http://tauri.localhost" "config" "--api_keys" "xGXURpGBdAdOxbswofwltmRc15Ab1tgZ", raw_out: false }
> [2025-06-20][21:26:15][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries/engines" to "/home/yopi/.config/Jan/data/engines"
> [2025-06-20][21:26:15][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries/engines/llama.cpp" to "/home/yopi/.config/Jan/data/engines/llama.cpp"
> [2025-06-20][21:26:15][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx-x64" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-avx-x64"
> [2025-06-20][21:26:15][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx-x64/b5509" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-avx-x64/b5509"
> [2025-06-20][21:26:15][app_lib::core::setup][INFO] Cortex server spawned successfully. PID: 115011
> [2025-06-20][21:26:15][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-cuda-cu11.7-x64" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-avx2-cuda-cu11.7-x64"
> [2025-06-20][21:26:15][app_lib::core::setup][INFO] [Cortex Terminated]: Signal Some(15), Code None
> [2025-06-20][21:26:15][app_lib::core::setup][INFO] Cleared child process lock after termination.
> [2025-06-20][21:26:15][app_lib::core::setup][WARN] Cortex server terminated unexpectedly.
> [2025-06-20][21:26:15][app_lib::core::setup][INFO] Waiting 5000ms before attempting restart 1/5...
> [2025-06-20][21:26:15][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-cuda-cu11.7-x64/b5509" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-avx2-cuda-cu11.7-x64/b5509"
> [2025-06-20][21:26:16][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-cuda-cu12.0-x64" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-avx2-cuda-cu12.0-x64"
> [2025-06-20][21:26:16][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-cuda-cu12.0-x64/b5509" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-avx2-cuda-cu12.0-x64/b5509"
> [2025-06-20][21:26:16][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-x64" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-avx2-x64"
> [2025-06-20][21:26:16][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-x64/b5509" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-avx2-x64/b5509"
> [2025-06-20][21:26:16][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx512-x64" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-avx512-x64"
> [2025-06-20][21:26:16][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx512-x64/b5509" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-avx512-x64/b5509"
> [2025-06-20][21:26:17][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-cuda-cu11.7-x64" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-noavx-cuda-cu11.7-x64"
> [2025-06-20][21:26:17][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-cuda-cu11.7-x64/b5509" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-noavx-cuda-cu11.7-x64/b5509"
> [2025-06-20][21:26:17][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-cuda-cu12.0-x64" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-noavx-cuda-cu12.0-x64"
> [2025-06-20][21:26:17][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-cuda-cu12.0-x64/b5509" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-noavx-cuda-cu12.0-x64/b5509"
> [2025-06-20][21:26:18][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-x64" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-noavx-x64"
> [2025-06-20][21:26:18][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-x64/b5509" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-noavx-x64/b5509"
> [2025-06-20][21:26:18][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries/engines/llama.cpp/linux-vulkan-x64" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-vulkan-x64"
> [2025-06-20][21:26:18][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries/engines/llama.cpp/linux-vulkan-x64/b5509" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-vulkan-x64/b5509"
> [2025-06-20][21:26:19][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/resources" to "/home/yopi/.config/Jan/data"
> [2025-06-20][21:26:19][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/resources/pre-install" to "/home/yopi/.config/Jan/data/pre-install"
> Could not create default EGL display: EGL_BAD_PARAMETER. Aborting...
> [2025-06-20][21:26:20][app_lib::core::setup][INFO] Spawning cortex-server (Attempt 2/5)
> [2025-06-20][21:26:20][app_lib::core::setup][DEBUG] Sidecar command: Command { cmd: LD_LIBRARY_PATH="/tmp/.mount_Jan_0.xnvX1c/usr/lib/:/tmp/.mount_Jan_0.xnvX1c/usr/lib/i386-linux-gnu/:/tmp/.mount_Jan_0.xnvX1c/usr/lib/x86_64-linux-gnu/:/tmp/.mount_Jan_0.xnvX1c/usr/lib32/:/tmp/.mount_Jan_0.xnvX1c/usr/lib64/:/tmp/.mount_Jan_0.xnvX1c/lib/:/tmp/.mount_Jan_0.xnvX1c/lib/i386-linux-gnu/:/tmp/.mount_Jan_0.xnvX1c/lib/x86_64-linux-gnu/:/tmp/.mount_Jan_0.xnvX1c/lib32/:/tmp/.mount_Jan_0.xnvX1c/lib64/::/tmp/.mount_Jan_0.xnvX1c/usr/lib/Jan/binaries" "/tmp/.mount_Jan_0.xnvX1c/usr/bin/cortex-server" "--start-server" "--port" "39291" "--config_file_path" "/home/yopi/.config/Jan/data/.janrc" "--data_folder_path" "/home/yopi/.config/Jan/data" "--cors" "ON" "--allowed_origins" "http://localhost:3000,http://localhost:1420,tauri://localhost,http://tauri.localhost" "config" "--api_keys" "xGXURpGBdAdOxbswofwltmRc15Ab1tgZ", raw_out: false }
> [2025-06-20][21:26:20][app_lib::core::setup][INFO] Cortex server spawned successfully. PID: 115080
> [2025-06-20][21:26:20][app_lib::core::setup][INFO] Cortex server started successfully, resetting restart count from 1 to 0.

@LFd3v Yes i did,nothing changed,like at all.

Here i tried environment variables : 
```
 WEBKIT_DISABLE_COMPOSITING_MODE=1 WEBKIT_DISABLE_DMABUF_RENDERER=1 ./Jan_0.6.1_amd64.AppImage
[2025-06-20][21:48:10][app_lib::core::setup][INFO] Installing extensions. Clean up: false, Stored version: 0.6.1, App version: 0.6.1
[2025-06-20][21:48:10][app_lib::core::setup][ERROR] Failed to run mcp commands: Failed to read config file: No such file or directory (os error 2)
[2025-06-20][21:48:10][app_lib::core::setup][INFO] Clean up function executed, sidecar processes killed.
[2025-06-20][21:48:10][app_lib::core::setup][INFO] Spawning cortex-server (Attempt 1/5)
[2025-06-20][21:48:10][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries" to "/home/yopi/.config/Jan/data"
[2025-06-20][21:48:10][app_lib::core::setup][DEBUG] Sidecar command: Command { cmd: LD_LIBRARY_PATH="/tmp/.mount_Jan_0.Chqkkx/usr/lib/:/tmp/.mount_Jan_0.Chqkkx/usr/lib/i386-linux-gnu/:/tmp/.mount_Jan_0.Chqkkx/usr/lib/x86_64-linux-gnu/:/tmp/.mount_Jan_0.Chqkkx/usr/lib32/:/tmp/.mount_Jan_0.Chqkkx/usr/lib64/:/tmp/.mount_Jan_0.Chqkkx/lib/:/tmp/.mount_Jan_0.Chqkkx/lib/i386-linux-gnu/:/tmp/.mount_Jan_0.Chqkkx/lib/x86_64-linux-gnu/:/tmp/.mount_Jan_0.Chqkkx/lib32/:/tmp/.mount_Jan_0.Chqkkx/lib64/::/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries" "/tmp/.mount_Jan_0.Chqkkx/usr/bin/cortex-server" "--start-server" "--port" "39291" "--config_file_path" "/home/yopi/.config/Jan/data/.janrc" "--data_folder_path" "/home/yopi/.config/Jan/data" "--cors" "ON" "--allowed_origins" "http://localhost:3000,http://localhost:1420,tauri://localhost,http://tauri.localhost" "config" "--api_keys" "ZqiOoHW1aQSiLFyi2QeHPxsb1W4n8R1N", raw_out: false }
[2025-06-20][21:48:10][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries/engines" to "/home/yopi/.config/Jan/data/engines"
[2025-06-20][21:48:10][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries/engines/llama.cpp" to "/home/yopi/.config/Jan/data/engines/llama.cpp"
[2025-06-20][21:48:10][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx-x64" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-avx-x64"
[2025-06-20][21:48:10][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx-x64/b5509" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-avx-x64/b5509"
[2025-06-20][21:48:10][app_lib::core::setup][INFO] Cortex server spawned successfully. PID: 118822
[2025-06-20][21:48:10][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-cuda-cu11.7-x64" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-avx2-cuda-cu11.7-x64"
[2025-06-20][21:48:10][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-cuda-cu11.7-x64/b5509" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-avx2-cuda-cu11.7-x64/b5509"
[2025-06-20][21:48:10][app_lib::core::setup][INFO] [Cortex Terminated]: Signal Some(15), Code None
[2025-06-20][21:48:10][app_lib::core::setup][INFO] Cleared child process lock after termination.
[2025-06-20][21:48:10][app_lib::core::setup][WARN] Cortex server terminated unexpectedly.
[2025-06-20][21:48:10][app_lib::core::setup][INFO] Waiting 5000ms before attempting restart 1/5...
[2025-06-20][21:48:10][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-cuda-cu12.0-x64" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-avx2-cuda-cu12.0-x64"
[2025-06-20][21:48:10][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-cuda-cu12.0-x64/b5509" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-avx2-cuda-cu12.0-x64/b5509"
[2025-06-20][21:48:15][app_lib::core::setup][INFO] Spawning cortex-server (Attempt 2/5)
[2025-06-20][21:48:15][app_lib::core::setup][DEBUG] Sidecar command: Command { cmd: LD_LIBRARY_PATH="/tmp/.mount_Jan_0.Chqkkx/usr/lib/:/tmp/.mount_Jan_0.Chqkkx/usr/lib/i386-linux-gnu/:/tmp/.mount_Jan_0.Chqkkx/usr/lib/x86_64-linux-gnu/:/tmp/.mount_Jan_0.Chqkkx/usr/lib32/:/tmp/.mount_Jan_0.Chqkkx/usr/lib64/:/tmp/.mount_Jan_0.Chqkkx/lib/:/tmp/.mount_Jan_0.Chqkkx/lib/i386-linux-gnu/:/tmp/.mount_Jan_0.Chqkkx/lib/x86_64-linux-gnu/:/tmp/.mount_Jan_0.Chqkkx/lib32/:/tmp/.mount_Jan_0.Chqkkx/lib64/::/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries" "/tmp/.mount_Jan_0.Chqkkx/usr/bin/cortex-server" "--start-server" "--port" "39291" "--config_file_path" "/home/yopi/.config/Jan/data/.janrc" "--data_folder_path" "/home/yopi/.config/Jan/data" "--cors" "ON" "--allowed_origins" "http://localhost:3000,http://localhost:1420,tauri://localhost,http://tauri.localhost" "config" "--api_keys" "ZqiOoHW1aQSiLFyi2QeHPxsb1W4n8R1N", raw_out: false }
[2025-06-20][21:48:15][app_lib::core::setup][INFO] Cortex server spawned successfully. PID: 118839
[2025-06-20][21:48:15][app_lib::core::setup][INFO] Cortex server started successfully, resetting restart count from 1 to 0.
[2025-06-20][21:48:17][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-x64" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-avx2-x64"
[2025-06-20][21:48:17][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx2-x64/b5509" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-avx2-x64/b5509"
[2025-06-20][21:48:17][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx512-x64" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-avx512-x64"
[2025-06-20][21:48:17][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries/engines/llama.cpp/linux-avx512-x64/b5509" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-avx512-x64/b5509"
[2025-06-20][21:48:17][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-cuda-cu11.7-x64" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-noavx-cuda-cu11.7-x64"
[2025-06-20][21:48:17][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-cuda-cu11.7-x64/b5509" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-noavx-cuda-cu11.7-x64/b5509"
[2025-06-20][21:48:18][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-cuda-cu12.0-x64" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-noavx-cuda-cu12.0-x64"
[2025-06-20][21:48:18][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-cuda-cu12.0-x64/b5509" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-noavx-cuda-cu12.0-x64/b5509"
[2025-06-20][21:48:27][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-x64" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-noavx-x64"
[2025-06-20][21:48:27][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries/engines/llama.cpp/linux-noavx-x64/b5509" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-noavx-x64/b5509"
[2025-06-20][21:48:27][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries/engines/llama.cpp/linux-vulkan-x64" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-vulkan-x64"
[2025-06-20][21:48:27][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/binaries/engines/llama.cpp/linux-vulkan-x64/b5509" to "/home/yopi/.config/Jan/data/engines/llama.cpp/linux-vulkan-x64/b5509"
[2025-06-20][21:48:33][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/resources" to "/home/yopi/.config/Jan/data"
[2025-06-20][21:48:33][app_lib::core::setup][INFO] Copying from "/tmp/.mount_Jan_0.Chqkkx/usr/lib/Jan/resources/pre-install" to "/home/yopi/.config/Jan/data/pre-install"
Could not create default EGL display: EGL_BAD_PARAMETER. Aborting...

```

--- a/.github/workflows/template-tauri-build-linux-x64.yml
+++ b/.github/workflows/template-tauri-build-linux-x64.yml
@@ -151,6 +151,12 @@ jobs:
           fi
       - name: Build app
         run: |
+          # Pin linuxdeploy version to prevent @tauri-apps/cli-linux-x64-gnu from pulling in an outdated version
+          TAURI_TOOLKIT_PATH="${XDG_CACHE_HOME:-$HOME/.cache}/tauri"
+          mkdir -p "$TAURI_TOOLKIT_PATH"
+          wget https://github.com/linuxdeploy/linuxdeploy/releases/download/1-alpha-20250213-2/linuxdeploy-x86_64.AppImage -O "$TAURI_TOOLKIT_PATH/linuxdeploy-x86_64.AppImage"
+          chmod +x "$TAURI_TOOLKIT_PATH/linuxdeploy-x86_64.AppImage"
+
           make build-tauri
           # Copy engines and bun to appimage
            wget https://github.com/AppImage/AppImageKit/releases/download/continuous/appimagetool-x86_64.AppImage -O ./appimagetool

